pkg/api/helpers.go

```diff
@@ -154,6 +154,17 @@ func IsStandardQuotaResourceName(str string) bool {
 	return standardQuotaResources.Has(str)
 }
 
+// AddStandardQuotaResourceNames add the resource names into the list of
+// resources known to the quota tracking system
+func AddStandardQuotaResourceNames(names ...ResourceName) {
+	for _, name := range names {
+		if len(strings.Split(string(name), "/")) != 2 {
```

Comment:
@derekwaynecarr ok, I will remove this validation... I felt weird about it anyway ;-)